mantidproject/mantid
# Add source code links in alg docs

Added source code links to the bottom of each algorithm documentation page

fixes #11821
Release note added - http://www.mantidproject.org/ReleaseNotes_3_5_UI_Changes#Documentation
### To test
1. Check the builds passed ok, especially the docs one, check the output from that one for any lines including "SourceLink"
2. possibly code review the sourcelink.py directive
3. Check a couple of C++ based algorithms pages and the links created, they should link to the exact code, specific to the commit
4. Check a python based algorithms pages and the links created, they should link to the exact code, specific to the commit. (Example: CollectHB3AExperimentInfo)
